package.json

```diff
@@ -16,7 +16,7 @@
     "sinon": "^1.17.6"
   },
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha tests/js/tests/*.test.js"
+    "test": "./node_modules/mocha/bin/mocha tests/js/tests/*.test.js --timeout 30000"
```

Comment:
See #12936:

> Error while running npm test:
>
> > Error: timeout of 15000ms exceeded. Ensure the `done()` callback is being called in this test.
>
> Happens very seldom, this is because of jsdom and integrating jQuery. Default timeout is 2000ms. And this is always not enough, so I increased it to be 15000ms.

I haven't seen this even once with Travis, only locally. I guess maybe it's because I work through Vagrant? Anyways increasing timeout to 30s solves the problem. I moved it here because jQuery is used everywhere (so jsdom needs to be used in every test file) to avoid repeating it in every test file.